src/main/java/org/example/validators/JobRoleValidator.java

```diff
@@ -0,0 +1,107 @@
+package org.example.validators;
+
+import org.example.exceptions.Entity;
+import org.example.exceptions.InvalidException;
+import org.example.models.JobRoleRequest;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Date;
+
+public class JobRoleValidator {
+
+    private static final int JOB_ROLE_MAX_LENGTH = 50;
+    private static final int JOB_ROLE_MIN_LENGTH = 5;
+
+    public void validateJobRole(final JobRoleRequest jobRoleRequest) throws
+            InvalidException {
+
+        if (jobRoleRequest.getRoleName() == null
+                || jobRoleRequest.getRoleName().isEmpty()) {
+            throw new InvalidException(Entity.JOB_ROLE,
+                    "Job Role Name is required.");
+        }
+        if (jobRoleRequest.getRoleName().length() > JOB_ROLE_MAX_LENGTH) {
+            throw new InvalidException(Entity.JOB_ROLE,
+                    "Job Role Name is greater than 50 characters.");
+        }
+        if (jobRoleRequest.getRoleName().length() < JOB_ROLE_MIN_LENGTH) {
+            throw new InvalidException(Entity.JOB_ROLE,
+                    "Job Role Name is less than 5 characters.");
+        }
+
+        if (jobRoleRequest.getDescription() == null
+                || jobRoleRequest.getDescription().isEmpty()) {
```

Comment:
Im being super picky now because i cant find anything wrong here... but you could also check for isBlank()... incase someone just enters a string of spaces only